examples/prometheus/prometheus.yaml

```diff
@@ -30,6 +30,15 @@ parameters:
   name: SESSION_SECRET
   generate: expression
   from: "[a-zA-Z0-9]{43}"
+- description: The threshold for the active build alert
```

Comment:
Do not add parameters here.  If you can't pick good defaults we shouldn't add the alert.

Reply:
Basically parameterization is the devil.  No parameters.  It's either a good rule, or it shouldn't be here (add it commented out).

Reply:
it seems to me the duration and volume of running builds that determines whether you have a problem or not is going to be determined in part by the size of your cluster.  (possibly also derived from steady state observations).  The fact that there aren't absolute values for it doesn't make it a bad rule.

if your suggestion is that such rules should be commented out rather than enabled out of the box, ok, but I would expect that's going to be the case for almost all our rules.  It's hard to imagine a metric who's health/sick condition *isn't* going to be determined in part by cluster characteristics.

Maybe what I need is a better understanding of the intent of this file?

Reply:
(and is there some other file where we should be putting in rules that we want for our actual online prod cluster?)